packages/flutter/lib/src/widgets/ticker_provider.dart

```diff
@@ -92,24 +109,73 @@ class TickerMode extends StatefulWidget {
   ///
   /// In the absence of a [TickerMode] widget, this function returns a
   /// [ValueListenable], whose [ValueListenable.value] is always true.
+  @Deprecated(
+    'Use TickerMode.getValuesNotifier to get both enabled and forceFrames. '
+    'This feature was deprecated after v3.35.0-0.0.pre.',
+  )
   static ValueListenable<bool> getNotifier(BuildContext context) {
     final _EffectiveTickerMode? widget = context
         .getInheritedWidgetOfExactType<_EffectiveTickerMode>();
     return widget?.notifier ?? const _ConstantValueListenable<bool>(true);
   }
 
+  /// Returns the effective ticker mode values for this subtree and establishes
+  /// a dependency on the surrounding [TickerMode].
+  static TickerModeData valuesOf(BuildContext context) {
+    final _EffectiveTickerMode? widget = context
+        .dependOnInheritedWidgetOfExactType<_EffectiveTickerMode>();
+    return widget?.values ?? const TickerModeData(enabled: true, forceFrames: false);
```

Comment:
`DefaultTextStyle` also has a similar behavior where it defaults to sensible values, [`DefaultTextStyle.fallback`](https://api.flutter.dev/flutter/widgets/DefaultTextStyle/DefaultTextStyle.fallback.html), if there's no ancestor `DefaultTextStyle`.

Since both `valuesOf` and `getValuesNotifier` has similar fallback logic, let's also introduce a `TickerModeData.fallback` member.